guide_drafts/trait_derives.md

```diff
@@ -0,0 +1,927 @@
+Deriving Traits in Depth
+===============
+
+Part of what makes Diesel's query builder so powerful is
+its ability to assist in writing safe SQL queries via rust.
+Diesel enables this level of safety through implementing 
+a series of traits on your structs.
+Writing these implementations by hand can be very laborious,
+so Diesel offers custom derives.
+
+In this guide,
+we will cover each trait in detail in terms of its use cases 
+and usage considerations.
+To make use of these derives,
+you will only need `#[macro_use] extern crate diesel;` at the root of your project.
```

Comment:
"To be able to use these derives, make sure you have …" sounds a bit more positive